.circleci/codebuild-checkout.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
```

Comment:
I recently learned that if we add a `-v` or `-x` we should get all the command logged explicitly https://stackoverflow.com/questions/2853803/how-to-echo-shell-commands-as-they-are-executed

Reply:
we do that sometimes when troubleshooting.